rocketpy/simulation/dispersion.py

```diff
@@ -695,8 +653,6 @@ def import_results(self, filename=None):
         self.import_inputs(filename=filepath)
         self.import_errors(filename=filepath)
 
-        return None
-
     def exportEllipsesToKML(
```

Comment:
Please consider renaming this method to follow the `snake_case` convention.  

`exportEllipsesToKML` --> `export_ellipses_to_kml`